package.json

```diff
@@ -269,7 +269,7 @@
     }
   ],
   "engines": {
-    "vscode": "^1.18.0"
+    "vscode": "*"
```

Comment:
As discussed let's not merge this until the feature moves into the latest stable build.